services/blue/src/api/DatabaseAccess/DatabaseExtensions.cs

```diff
@@ -6,6 +6,18 @@ namespace artiso.AdsdHotel.Blue.Api
 {
     internal static class DatabaseExtensions
     {
+        public static async ValueTask<IDbTransaction> BeginTransactionAsync(this IDbConnection connection)
```

Comment:
Why are we returning a ValueTask? Since the MySqlConnection inherits from DbConnection wouldn't this method complete asynchronous more often than not?

Reply:
I'm matching the [DbConnection signature](https://docs.microsoft.com/en-us/dotnet/api/system.data.common.dbconnection.begintransactionasync?view=net-5.0#System_Data_Common_DbConnection_BeginTransactionAsync_System_Threading_CancellationToken_):
```csharp
ValueTask<DbTransaction> BeginTransactionAsync(...
```